backend/src/main/java/de/bund/digitalservice/ris/norms/domain/entity/Norm.java

```diff
@@ -239,8 +239,10 @@ public boolean isInkraftAt(LocalDate date) {
 
     return (
       inkrafttreteDatum.isPresent() &&
-      inkrafttreteDatum.get().isBefore(date) &&
-      (ausserkrafttreteDatum.isEmpty() || ausserkrafttreteDatum.get().isAfter(date))
+      (inkrafttreteDatum.get().isBefore(date) || inkrafttreteDatum.get().isEqual(date)) &&
```

Comment:
What about:

```
    return (
      inkrafttreteDatum.isPresent() &&
      !inkrafttreteDatum.get().isAfter(date) &&
      (ausserkrafttreteDatum.isEmpty() || !ausserkrafttreteDatum.get().isBefore(date))
    );
```

Or even something like:
```
    return (
      inkrafttreteDatum.map(d -> !d.isAfter(date)).orElse(false) &&
      ausserkrafttreteDatum.map(d -> !d.isBefore(date)).orElse(true)
    );
```

I personally find "big" condition groups hard to read and therefore prefer always a more concise version of it. If you agree with me choose the one you can read better :)

Reply:
While i do read the first option better on first glance. The second one is more concise. Thanks for the optimisation.